spring-restdocs-core/src/test/java/org/springframework/restdocs/cli/ConcatenatingCommandFormatterTests.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2014-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.restdocs.cli;
+
+import java.util.Collections;
+
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Tests for {@link CommandFormatter}.
+ *
+ * @author Tomasz Kopczynski
+ */
+public class ConcatenatingCommandFormatterTests {
+
+	private CommandFormatter newlineCommandFormatter = ConcatenatingCommandFormatter.newlineSeparator();
+	private CommandFormatter spaceCommandFormatter = ConcatenatingCommandFormatter.spaceSeparator();
+	private CommandFormatter customCommandFormatter = ConcatenatingCommandFormatter.customSeparator("-");
+
+	private static final String STR = "test";
+
+	@Test
+	public void noElementsTest() {
```

Comment:
If the static methods are moved, this test could be simplified to just check that no elements results in an empty string. It would be good to have a similar test for `null` too.